app/src/debugUnitTests/java/com/artemzin/qualitymatters/developer_settings/DeveloperSettingsModelImplTest.java

```diff
@@ -33,8 +35,8 @@ public void beforeEachTest() {
                 developerSettings,
                 mock(OkHttpClient.class),
                 new HttpLoggingInterceptor(),
-                mock(LeakCanaryProxy.class)
-        );
+                mock(LeakCanaryProxy.class),
+                new Paperwork(mock(QualityMattersApp.class)));
```

Comment:
The tests fail because Paperwork receives a mock context, and cannot read the json file. One solution would be to turn this test into a Robolectric test.

Another option would be to remove test methods related to paperwork functionality, as there's no need to test its internal workings here. Everything is tested in the paperwork already.